services/entity-server/src/middleware/data-transforms.js

```diff
@@ -82,3 +82,17 @@ export const extractProjectList = data =>
     description: ideaContest.description,
     title: ideaContest.title
   }));
+
+export const sortResources = data => {
+  var dic = new Map();
+  for (var { predicate, object } of data) {
+    if (dic.has(predicate.value)) {
+      dic.get(predicate.value).objects.push(object);
+    } else {
+      dic.set(predicate.value, { predicate, objects: [object] });
+    }
+  }
```

Comment:
loops are slow, callbacks would be faster, in this case forEach, map even better
also this is not deep copying data, so technically it could be mutated

probably works though